src/App.jsx

```diff
@@ -30,6 +31,7 @@ export const App = () => {
     dispatch(getAllAreas());
     dispatch(getAllCategories());
     dispatch(getAllIngredients());
+    dispatch(getAllTestimonials());
```

Comment:
remove getAllTestimonials from app. this request should be in Testimonials component